packages/@sanity/core/src/commands/dataset/alias/createAliasHandler.js

```diff
@@ -0,0 +1,64 @@
+import promptForDatasetName from '../../../actions/dataset/datasetNamePrompt'
+import promptForDatasetAliasName from '../../../actions/dataset/alias/datasetAliasNamePrompt'
+import validateDatasetAliasName from '../../../actions/dataset/alias/validateDatasetAliasName'
+import validateDatasetName from '../../../actions/dataset/validateDatasetName'
+import * as aliasClient from './datasetAliasesClient'
+import {ALIAS_PREFIX} from './datasetAliasesClient'
+
+export default async (args, context) => {
+  const {apiClient, output, prompt} = context
+  const [, alias, targetDataset] = args.argsWithoutOptions
+  const client = apiClient()
+
+  const nameError = alias && validateDatasetAliasName(alias)
+  if (nameError) {
+    throw new Error(nameError)
+  }
+
+  const [datasets, aliases, projectFeatures] = await Promise.all([
+    client.datasets.list().then((sets) => sets.map((ds) => ds.name)),
+    aliasClient.listAliases(client).then((sets) => sets.map((ds) => ds.name)),
```

Comment:
Shouldn't we try to re-use the `client` and not importing and calling this directly?

Reply:
The handlers are just fine the way it is when the cli supports grouped commands.